app/forms/ImporterEoriFormProvider.scala

```diff
@@ -26,6 +26,11 @@ class ImporterEoriFormProvider @Inject() extends Mappings {
   def apply(): Form[EORI] =
     Form(
       "value" -> text("importerEori.error.required")
-        .verifying(maxLength(17, "importerEori.error.length")).transform[EORI](EORI.apply, _.value)
+        .verifying(minLength(14, "importerEori.error.length"))
+        .verifying(maxLength(14, "importerEori.error.length"))
```

Comment:
Hi @somaraja Should we not set the max to 17? I think the user could add an EORI with GB follwed by either 12 or 15 digits. Can we discuss this tomorrow?